modules/__tests__/TestSequences/Pop.js

```diff
@@ -0,0 +1,33 @@
+import expect from 'expect'
+import execSteps from './execSteps'
+
+export default (history, done) => {
+  const steps = [
+    (location) => {
+      expect(location).toMatch({
+        pathname: '/'
+      })
+
+      history.push('/home')
+    },
+    (location, action) => {
+      expect(action).toEqual('PUSH')
+      expect(location).toMatch({
+        pathname: '/home'
+      })
+
+      history.pop()
+    },
+    (location, action) => {
+      expect(action).toEqual('POP')
+      expect(location).toMatch({
+        pathname: '/'
+      })
+      expect(history.entries).toMatch([{
+        pathname: '/',
+      }])
```

Comment:
Why is this last assertion important? Is your app reading from `history.entries`? Why can't your app do what it needs to do at this point with just the current `history.location`?